precompile/contracts/nativeminter/nativemintertest/config_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
I moved these in to match what you did for the deployerallowlist. I think all of these folders need to be cleaned up / reorganized, but I would appreciate your guidance @ceyonur

Reply:
trying to put everything in the top level package also results in cyclic dependencies. 

Would you like to put a PR up into this PR to restructure the files? I'm not sure if I follow exactly what you are suggesting. It really is a mess...